keps/sig-network/20191205-kube-proxy-IP-node-binding.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Give the ability to the LoadBalancer provisioner or a user to disable the LB IP binding
+authors:
+  - "@Sh4d1"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cloud-provider
+reviewers:
+  - "@thockin"
+  - "@cheftako"
+  - TBD
+approvers:
+  - "@thockin"
+  - TBD
+editor: TBD
+creation-date: 2019-12-05
+last-updated: 2019-12-21
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Give the ability to the LoadBalancer provisioner or a user to disable the LB IP binding
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+<!-- /toc -->
+
+## Summary
+
+The different kube proxy implentations (at least ipvs and iptables) are today sort of binding the External IPs of the LoadBalancer Service to each node. (iptables is creating some rules to redirect directly the packets to the service and ipvs is binding the IP to one interface on the node). This feature is exisiting because:
+
+1. To redirect pod traffic that is intended for LoadBalancer IP directly to the service, bypassing the loadbalancer for path optimization.
+2. Some LBs are sending packet with the destination IP equal to the LB IP, so the IP needs to be routable directly to the right service (otherwise looping back and forth)
```

Comment:
2. Some LBs are sending packet with the destination IP equal to the LB IP, so the IP needs to be ~routable~ `routed` directly to the right service (otherwise looping back and forth)`.`